wallets/rn_cli_wallet/src/screens/Settings.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {Text, StatusBar, useColorScheme, View, StyleSheet} from 'react-native';
+
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+import '@walletconnect/react-native-compat';
```

Comment:
you could add this import [here](https://github.com/WalletConnect/react-native-examples/blob/d4332b08195b3d116185aec9384630c11f11f497/wallets/rn_cli_wallet/index.js#L5) and it remove from elsewhere